sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -481,35 +481,47 @@ private bool TryGetReturnTypePropertyBindings(ITypeSymbol returnTypeSymbol, bool
                         foundHttpOutput = true;
                         bindingsList.Add(GetHttpReturnBinding(prop.Name));
                     }
-                    else if (prop.GetAttributes().Length > 0) // check if this property has any attributes
+
+                    if (prop.GetAttributes().Length > 0)
```

Comment:
Store result of `prop.GetAttributes()` to a variable and use that in line 485 and 487.